packages/commonwealth/client/scripts/views/components/SublayoutHeader/useUserMenuItems.tsx

```diff
@@ -297,7 +299,14 @@ const useUserMenuItems = ({
       {
         type: 'default',
         label: 'My transactions',
-        onClick: () => navigate(`/myTransactions`, {}, null),
+        onClick: () =>
```

Comment:
shouldn't the label be updated to "My rewards" when `rewardsEnabled=true`?

Reply:
nope, this label is adequate, it takes user to specific tab/table in the /rewards page